doc/release/upcoming_changes/14510.compatibility.rst

```diff
@@ -1,10 +1,12 @@
-`numpy.lib.recfunctions.drop_fields` can no longer return ``None``
-------------------------------------------------------------------
+`numpy.lib.recfunctions.drop_fields` can no longer return `None`
+----------------------------------------------------------------
 If ``drop_fields`` is used to drop all fields, previously the array would
-be completely discarded and ``None`` returned. Now it returns an array of the
+be completely discarded and `None` returned. Now it returns an array of the
 same shape as the input, but with no fields. The old behavior can be retained
```

Comment:
Was this note really broken?

Reply:
The note was fine. The issue is that when the template consumes it, no blank line was added after the note. That meant the code block had no blank line after it, which sphinx RST doesn't like.

Reply:
OK, I think I like the solution of just adding a bit of text under the fragment best. Feels a bit wrong that there is no better way, but if we like the autmatic addition of the link it is a bit tricky (unless we remove automatic whitespace stripping, but...). Did not realize `None` was fine with single quotes.